imxIcons/domain/helpers.py

```diff
@@ -0,0 +1,12 @@
+from imxIcons.iconEntity import IconEntity
+
+
+def copy_add_new_imx_path(
```

Comment:
Ah, sorry op mijn telefoon is dit lastig te zien :)

Reply:
Copy_and_assign_imx_path is misschien een betere naam?

Reply:
goed voorstel, doe ik